commit 6497d1f8875cb7e3007f714336cc09c06fed235b
Author: Jia Tan <jiat0218@gmail.com>
Date:   Tue Jul 25 20:14:53 2023 +0800

    CI: Windows CI rename system matrix variable -> msys2_env.
    
    Calling the MSYS2 environment "system" was a bit vague and should be
    more specific.

diff --git a/.github/workflows/windows-ci.yml b/.github/workflows/windows-ci.yml
index 1b018479..c89eb605 100644
--- a/.github/workflows/windows-ci.yml
+++ b/.github/workflows/windows-ci.yml
@@ -20,12 +20,7 @@ jobs:
         # them and we want to ensure that we support all potential users.
         # clang64 currently fails due to a warning, so support will be
         # added later.
-        system:
-          - mingw64
-          - mingw32
-          - ucrt64
-          - clang64
-          - msys
+        msys2_env: [mingw64, mingw32, ucrt64, clang64, msys]
 
     # Set the shell to be msys2 as a default to avoid setting it for
     # every individual run command.
@@ -44,7 +39,7 @@ jobs:
       - name: Setup MSYS2
         uses: msys2/setup-msys2@v2
         with:
-          msystem: ${{ matrix.system }}
+          msystem: ${{ matrix.msys2_env }}
           update: true
           install: pactoys
 
@@ -70,11 +65,11 @@ jobs:
         # the packages possible in matrix setup without a burdensome amount
         # of ifs.
       - name: Install Dependencies
-        if: ${{ matrix.system == 'msys' }}
+        if: ${{ matrix.msys2_env == 'msys' }}
         run: pacman --noconfirm -S --needed autotools base-devel doxygen gettext-devel gcc
 
       - name: Install Dependencies
-        if: ${{ matrix.system != 'msys' }}
+        if: ${{ matrix.msys2_env != 'msys' }}
         run: pacboy --noconfirm -S --needed autotools:p gcc:p clang:p make:p doxygen:p
 
 
@@ -116,5 +111,5 @@ jobs:
       - uses: actions/upload-artifact@v3
         if: ${{ failure() }}
         with:
-          name: ${{ matrix.os }} ${{ matrix.build_system }} Test Logs
+          name: ${{ matrix.msys2_env }} ${{ matrix.build_system }} Test Logs
           path: build-aux/artifacts